Orange/data/variable.py

```diff
@@ -451,8 +452,14 @@ def __reduce__(self):
         # Use make to unpickle variables.
         return make_variable, (self.__class__, self._compute_value, self.name), self.__dict__
 
-    def copy(self, compute_value):
-        var = type(self)(self.name, compute_value=compute_value, sparse=self.sparse)
+    def rename(self, name):
+        return self._copy(name, self.compute_value)
+
+    def copy(self, compute_value=None):
```

Comment:
Perhaps we should just have `def copy(self, compute_value=None, *, name=None)`, and then, of course, `if name is None: name = self.name`. So we avoid superfluous `_copy` function.